Common/TableProducer/PID/pidTOFTrackTime.cxx

```diff
@@ -0,0 +1,269 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   pidTOFTrackTime.cxx
+/// \author Nicolo' Jacazio
+/// \brief  Task to convert the trackTime to the tofSignal.
+///
+
+// O2 includes
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/RunningWorkflowInfo.h"
+#include "ReconstructionDataFormats/Track.h"
+#include <CCDB/BasicCCDBManager.h>
+#include "Common/Core/PID/PIDResponse.h"
+#include "Common/Core/PID/PIDTOF.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Common/Core/TrackSelection.h"
+#include "Common/Core/TrackSelectionDefaults.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::pid;
+using namespace o2::pid::tof;
+using namespace o2::framework::expressions;
+using namespace o2::track;
+
+void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
+{
+  std::vector<ConfigParamSpec> options{{"add-qa", VariantType::Int, 0, {"Produce TOF PID QA histograms"}}};
+  std::swap(workflowOptions, options);
+}
+
+#include "Framework/runDataProcessing.h"
+
+struct trackTime {
```

Comment:
When this is moved to the pidFull task, this executable can be renamed to contain qa